jit_maker/src/jit_maker.py

```diff
@@ -73,6 +71,8 @@ def __init__(
         self.sub_accounts: list[int] = config.sub_accounts  # type: ignore
         self.market_indexes: list[int] = config.market_indexes  # type: ignore
         self.market_type = config.market_type
+        self.target_leverage = config.target_leverage
+        self.spread = config.spread
```

Comment:
make these optional with default values of 1 and 0 for target_leverage and spread?